Userland/Libraries/LibJS/Runtime/Intl/DurationFormatPrototype.cpp

```diff
@@ -45,8 +45,8 @@ JS_DEFINE_NATIVE_FUNCTION(DurationFormatPrototype::format)
     if (!is_valid_duration_record(record))
         return vm.throw_completion<RangeError>(ErrorType::TemporalInvalidDurationLikeObject);
 
-    // 5. Let formatted be ? PartitionDurationFormatPattern(df, record).
-    auto formatted = TRY(partition_duration_format_pattern(vm, *duration_format, record));
+    // 5. Let formatted be ! PartitionDurationFormatPattern(df, record).
+    auto formatted = partition_duration_format_pattern(vm, *duration_format, record);
```

Comment:
Not sure about 402, but they don't do the `!` anymore in upstream 262 for AOs that don't return a completion (like this one)